ls1intum/Artemis
# [Chore] - strict template check fixes

### Checklist

### Motivation and Context


We want to integrate strict template checks and need many fixes for that

### Description

- Fixed some bugs in learning-goal-card.component.ts and course-registration-selector.component.html

### Steps for Testing


1. Log in to Artemis
2. Try to add a student to a course -> should have no abnormal behaviour
3. Access learning goals in the student course statistics -> should have no abnormal behaviour
